scripts/globals/mobskills/bone_crunch.lua

```diff
@@ -1,29 +1,29 @@
 ---------------------------------------------
---  Bone Crunch
---
---  Description: plague
+-- Bone Crunch
+-- Deals damage to a single target. Additional effect: Plague
 ---------------------------------------------
-
 require("scripts/globals/settings")
 require("scripts/globals/status")
 require("scripts/globals/monstertpmoves")
-
 ---------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
     return 0
 end
 
 function onMobWeaponSkill(target, mob, skill)
-    local typeEffect = dsp.effect.PLAGUE
-
-    MobPhysicalStatusEffectMove(mob, target, skill, typeEffect, 5, 0, 30)
 
     local numhits = 1
     local accmod = 1
     local dmgmod = 2.5
     local info = MobPhysicalMove(mob,target,skill,numhits,accmod,dmgmod,TP_NO_EFFECT)
     local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,dsp.attackType.PHYSICAL,dsp.damageType.BLUNT,info.hitslanded)
+    local typeEffect = dsp.effect.PLAGUE
```

Comment:
no need for a variable since it is never changing and typeEffect is only called once, just use it directly

Reply:
Good deal 👍

Reply:
same on the other scripts, guess I should have said so in the first place